### DISCUSSION TITLE
Language localization: Spanish

### DISCUSSION THREAD

**Discussion by elmau** [mozilla/hubs#3108] [General Questions & Discussion] [answered]
Hello, we made a localization file for Spanish.

Download it from here:

https://drive.google.com/file/d/1XiXHxfFwZdEDsfMgzigjpjAj3tcv0e5r/view?usp=sharing

We also noticed that some phrases in the UI doesn't exist in the en.json file. We don't know the code names, but we took notes of where they are in the UI. Please, contact me to complete the missing lines. Thank you.

**Comment by robinkwilson** [comment, marked as answer]
Thank you! We created a PR so this can be looked over and merged into hubs: https://github.com/mozilla/hubs/pull/3146

We really appreciate helping us out with the translations!

**Reply by elmau** [reply]
We'll be working in a translation to russian soon.

**Reply by johnshaughnessy** [reply]
@elmau Someone seems to have submited Russian translations yesterday
https://github.com/mozilla/hubs/discussions/3174
